cmd/node/main.go

```diff
@@ -137,13 +138,17 @@ func main() {
 
 	// Set up the main API server
 	routes := []api.Route{
+		// Node routes
 		api.NewVersionHandler(),
 		controller.NewGetNodeRoute(db.Manager),
 		controller.NewLoginRoute(&controller.PDSClient{NodeConfig: nodeConfig}),
 		controller.NewAddUserRoute(nodeCtrl),
 		controller.NewInstallAppRoute(nodeCtrl),
 		controller.NewStartProcessHandler(nodeCtrl),
 		controller.NewMigrationRoute(nodeCtrl),
+
+		// App store routes
+		appstore.NewAvailableAppsRoute(nodeConfig),
```

Comment:
I wonder if we should refactor this so that instead we pass in some function like `registerRoute` which calls `router.Handle` under the hood to all of the down streams, that they can use.

In the long term, it seems annoying to have to add every single route into `main.go`. We probably want to do this for each component inside of it's entrypoint, like in a central place for controller, appstore etc. so that if you wanna update app store, you don't need to update main.go

Reply:
Yeah I agree, I can note that for the future.